source/documentation/appendix-docs/conda.md

```diff
@@ -139,11 +114,24 @@ environment.
 conda env update -f environment.yml --prune
 ```
 
-[packrat]: https://rstudio.github.io/packrat/
-[anaconda search]: https://anaconda.org/search
-[conda cheat sheet]:
-  https://conda.io/projects/conda/en/latest/user-guide/cheatsheet.html
 
-```{r global_options, include=FALSE}
-{knitr::opts_chunk$set(eval = FALSE)}
+## Shiny
+To use conda in shiny applications, you will need to use a different dockerfile to deploy the app. The [conda branch of the rshiny-template](https://github.com/moj-analytical-services/rshiny-template/tree/conda) has an appropriate dockerfile for this purpose. This is also necessary if you wish to use Python in your Shiny application, including using `dbtools` for accessing Athena databases.
+
+## Platform limitations
+
+There are a number of limitations and pitfalls to conda management to be aware of.
+
+### R package  versions on conda
```

Comment:
Are there two spaces between 'package' and 'versions'?

Reply:
Not any more!